### PR TITLE
LPS-137323 Close Info Panel, whenever a page is changed in Business Info - Info Panel

### PR DESCRIPTION
**Pull request by jairmedeiros**
- LPS-137323 Close Info Panel by Step's changes
- LPS-137323 Close Info Panel by Dismiss button

### CODE REVIEW

**Review by kevenleone — APPROVED** [review]
Looks good to me 🔥 

I noted, in some points of the code, the missing of alignment and indentation, but we're really in few parts.

We can fix the alignment later 😄

File: modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/hooks/useStepWizard.js [hunk 1]
```diff
@@ -4,18 +4,27 @@ import { useWatch } from "react-hook-form";
...
+  const [dispatchEvent] = useCustomEvent(TIP_EVENT);
   const { state, dispatch } = useContext(AppContext);
 
   useEffect(() => {
     _updateStepPercentage();
   }, [form]);
 
+  useEffect(() => {
+    dispatchEvent({
+			hide: true,
```
Comment (kevenleone) [inline modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/hooks/useStepWizard.js:24]: Here

File: modules/apps/site/site-insurance-site-initializer/src/main/resources/com/liferay/site/insurance/site/initializer/internal/dependencies/ddm-templates/tip-card/ddm_template.ftl [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+	#tip {
 		background-color: #F9F9F9;
 		padding: 24px;
 		width: auto;
 	}
+
+	#tip.hide {
+		opacity: 0;
+		transition: opacity .5s linear;
+    	pointer-events: none;
```
Comment (kevenleone) [inline modules/apps/site/site-insurance-site-initializer/src/main/resources/com/liferay/site/insurance/site/initializer/internal/dependencies/ddm-templates/tip-card/ddm_template.ftl:11]: here

File: modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/hooks/useStepWizard.js (hunk 1 above)
Comment (jairmedeiros) [reply modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/hooks/useStepWizard.js]: done.

File: modules/apps/site/site-insurance-site-initializer/src/main/resources/com/liferay/site/insurance/site/initializer/internal/dependencies/ddm-templates/tip-card/ddm_template.ftl (hunk 2 above)
Comment (jairmedeiros) [reply modules/apps/site/site-insurance-site-initializer/src/main/resources/com/liferay/site/insurance/site/initializer/internal/dependencies/ddm-templates/tip-card/ddm_template.ftl]: done.

**Comment by kevenleone** [comment]
Thanks @jairmedeiros !!!

**Pull request merged**